src/StatusQ/Components/StatusTagSelector.qml

```diff
@@ -355,8 +355,7 @@ Item {
                     anchors.rightMargin: 8
                     anchors.leftMargin: 8
                     radius: 8
-                    visible: (root.sortedList.count > 0)
-                    color: (userListView.currentIndex === index) ? Theme.palette.baseColor2 : "transparent"
+                    color: sensor.containsMouse ? Theme.palette.baseColor2 : "transparent"
```

Comment:
hmm is this working when navigating with the up/down arrows? I'd rather set the current index to index when the mouse area 'contains mouse'

Reply:
This fixes the issue but I agree with @alexandraB99 that the complete fix should be:
```qml
  visible: wrapper.ListView.isCurrentItem
  color: Theme.palette.baseColor2
```

Reply:
@borismelnik also the fix won't be completed without fixing `userListView.decrementCurrentIndex()` and `userListView.incrementCurrentIndex()` calls above. We use a visual workaround to hide elements instead of removing them from the data model. Therefore the increment/decrement calls don't take this into considerations and `currentIndex` can be one of the invisible items.